rubocop.yml

```diff
@@ -1,4 +1,48 @@
 # LTV Rubocop config
+#
+# This is a living config; if a rule doesn't make sense, configure it or
+# disable it to align with our style guidelines.
+#
+# Documentation on specific cops can be found in the Rubocop docs:
+#
+#   https://docs.rubocop.org/rubocop/index.html
 
+inherit_mode:
+  merge:
+    - Exclude
+AllCops:
+  NewCops: disable
+  Exclude:
+    - 'bin/{rails,rake,resque,rspec,spring}'
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'config/unicorn.rb'
 Layout/LineLength:
   Max: 100
+Lint/AmbiguousBlockAssociation:
+  Exclude:
+    - "spec/**/*"
+Metrics/BlockLength:
+  Exclude:
+    - 'Rakefile'
+    - '**/*.rake'
+    - 'spec/**/*.rb'
+Style/BlockComments:
+  Exclude:
+    - 'db/migrations/*'
+Style/Documentation:
+  Exclude:
+    - 'db/migrations/*'
+Style/FrozenStringLiteralComment:
+  Exclude:
+    - 'db/migrations/*'
+Style/StringLiterals:
+  Enabled: false
+Style/PercentLiteralDelimiters:
+  PreferredDelimiters:
+    default: "()"
+    "%i": "()"
+    "%I": "()"
+    "%r": "()"
+    "%w": "()"
+    "%W": "()"
```

Comment:
Personally, I'm not a fan of these rules. I put them in the Core Rubocop config because it matches the [old style guide](https://github.com/ltvco/ruby), but I feel like I see [the default Rubocop style](https://github.com/rubocop/ruby-style-guide#percent-literal-braces) used more commonly in Ruby. What are your thoughts?

Reply:
Hmmm...I like standardizing them, but I don't have a dog in this fight. I think maybe someone who isn't me might be better suited to decide what our in-house linting styles should be. I tend to think our linting should adhere to our style guide, but if that's in flux, then I guess it's open season.